nilearn/datasets/description/main.rst

```diff
@@ -0,0 +1,37 @@
+Montreal Artificial Intelligence and Neuroscience conference 2018 datasets
+
+
+Notes
+-----
+This functional MRI datasets is used for teaching how to use
+machine learning to predict age from rs-fmri with Nilearn.
+
+The dataset consists of 50 children (ages 3-13) and 33 young adults (ages
+18-39). This rs-fmri data can be used to try to predict who are adults and
+who are children.
+
+The data is downsampled to 4mm resolution for convenience. The original
+data is downloaded from OpenNeuro.
+
+Here: https://openneuro.org/datasets/ds000228/versions/1.0.0
+
+Track issue for more information:
+https://github.com/nilearn/nilearn/issues/1864
```

Comment:
This will be closed once this PR is merged, no ? Is it still useful to link, here ?